Formula/kns.rb

```diff
@@ -1,15 +1,17 @@
 class Kns < Formula
   desc "quick Kubernetes Namespace Switcher"
```

Comment:
>Quick Kubernetes namespace **and context** switcher

Reply:
💡

Reply:
"kns" is just a namespace switcher. 😄
It should indeed mention the ktx tool as well somewhere.